index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Graffiti bootcamp - project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway:300,400" rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+</head>
+
+<body>
+  <header>
+    <!-- video or image as a header is cool :) -->
+    <div id=hero>
+      <video autoplay loop muted preload="auto" id="graffitiVideo">
+        <source src="./videos/compressed_graffiti.mp4" type="video/mp4">
+        Your browser doesn't support HTML5 video.
+      </video>
+      <div class="content">
+        <h1>Graffiti artist bootcamp</h1>
+        <h2> Want to become a real graffiti artist ? Sign up to the graffit artist bootcamp this fall 2021.</h2>
+        <p><button id=signbutton class="w3-button w3-black w3-round-xxlarge">Sign up to Graffiti bootcamp</button></p>
+      </div>
+    </div>
+  </header>
+  <div class="content-container">
+    <!-- Signup form -->
+    <div class="form-group">
+      <form class="form" action="http://httpbin.org/anything" method="POST">
+        <div class="form-element">
+          <h2>Sign up to bootcamp</h2>
+        </div>
+        <div class="form-element">
+          <label for="fname">First name:</label><br>
+          <input name="firstname" id="fname" type="text" placeholder="firstname" required>
+        </div>
+        <div class="form-element">
+          <label for="lastname">Last name:</label><br>
+          <input name="lname" id="lname" type="text" placeholder="lastname" required>
+        </div>
```

Comment:
I assume you change `fname` and `lname` to differentiate between the name/id and for, but consistency is preferred in this case to avoid thinking "Which convention did I choose for id/name?"

Here, the `for="lastname"` does not match `id="lname"`